src/components/keyStats.test.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import * as redux from "react-redux";
+import {
+  render,
+  cleanup,
+  screen,
+  prettyDOM,
+  waitForElement,
+  fireEvent,
+} from "@testing-library/react";
+import App from "../App";
+import { KeyStats } from "./keyStats";
+import { useQuoteSelector } from "./componentHooks/useCompanySelector";
+import { useStatSelector } from "./componentHooks/useTickerSelector";
+
+jest.mock("./componentHooks/useQuoteSelector", () => ({
+  useQuoteSelector: () => {
+    return {
+     previousClose: 100,
+     low: 50,
+     high: 200,
+     latestVolume: 100,
+     marketCap: 1000,
+     open: 75,
+     week52Low: 30,
+     week52High: 250,
+     avgTotalVolume: 150
+        }
+    },
+}));
+jest.mock("./componentHooks/useStatsSelector", () => ({
+  useStatsSelector: () => {
+    return {
+        peRatio: 10,
+        earningsPerShare: 1,
+        dividendYield: 0.5
+    }
+  },
+}));
+
+describe("Key Stats component", () => {
+  let app;
+
+  beforeEach(async () => {
+    const { container } = render(<KeyStats />);
+    app = container;
+    
+    await waitForElement(() => screen.getByText("Previous Close"));
+    console.log(prettyDOM(app));
+  });
+
+  afterEach(cleanup);
+
+  test("Key stats data is rendered", () => {
+    const keyStatsHeader = screen.getByText("KEY STATS");
```

Comment:
Just a general comment on tests like these; if you don't use the const's name anywhere except in the following line, it might make more sense to inline the screen.getByText, just to cut down on the line count. So, 

`const previousCloseText = screen.getByText("Previous Close");
expect(previousCloseText).toBeInTheDocument();`

could be 

`expect(screen.getByText("Previous Close")).toBeInTheDocument();`

Reply:
fair enough